hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -32,160 +47,193 @@ import Hydra.Node (
   queryLedgerState,
   runHydraNode,
  )
-import System.Timeout (timeout)
-import Test.Hspec (
-  Spec,
-  describe,
-  expectationFailure,
-  it,
-  shouldContain,
-  shouldNotBe,
-  shouldReturn,
- )
+import Test.Hspec (Spec, describe, it)
+import Test.Util (failAfter, failure, shouldContain, shouldNotBe, shouldReturn, shouldRunInSim)
 
 spec :: Spec
 spec = describe "Behavior of one ore more hydra-nodes" $ do
   describe "Sanity tests of test suite" $ do
-    it "is Ready when started" $ do
-      n <- simulatedChainAndNetwork >>= startHydraNode 1
-      queryNodeState n `shouldReturn` Ready
-
-    it "is NotReady when stopped" $ do
-      n <- simulatedChainAndNetwork >>= startHydraNode 1
-      stopHydraNode n
-      queryNodeState n `shouldReturn` NotReady
-
-  describe "Hydra node integration" $ do
-    it "accepts Init command" $ do
-      n <- simulatedChainAndNetwork >>= startHydraNode 1
-      sendRequest n (Init [1]) `shouldReturn` ()
-
-    it "accepts Commit after successful Init" $ do
-      n <- simulatedChainAndNetwork >>= startHydraNode 1
-      sendRequest n (Init [1])
-      sendRequest n (Commit 1)
-
-    it "accepts a tx after the head was opened between two nodes" $ do
+    it "is Ready when started" $
+      shouldRunInSim $ do
```

Comment:
Given that _all_ tests use `shouldRunInSim` perhaps we could have a specialised `it` combinator for that?

Reply:
Thought about that quickly, but then put it off as being about the same overhead in typing/reading. I have no preference..what would you call that `it` variation?